embedded-graphics/src/primitives/line.rs

```diff
@@ -54,7 +54,7 @@ impl Dimensions for Line {
     }
 
     fn bottom_right(&self) -> Point {
-        self.top_left() + self.size()
+        self.top_left() + (self.size() - Size::new(1, 1))
```

Comment:
Will this off-by-one be resolved with the other work around #182?

Reply:
Subtracting `(1, 1)` should make this consistent with #182.